src/Tutorial/PatientBrowser/index.html

```diff
@@ -0,0 +1,288 @@
+            span.setAttribute('id', 'Span_Patient_'+updatedNum);
+            span.innerHTML = " View";
+            button.appendChild(span);
+            cellInstruction.appendChild(button);
+            button.setAttribute('id', 'View_Patient_'+updatedNum);
+            // counter.innerHTML = i+1;
+
+            
+            lastName.innerHTML = JSON.stringify(json.entry[i].resource.name[0].family).replace(/['"]+/g, '');
+            firstName.innerHTML = JSON.stringify(json.entry[i].resource.name[0].given[0]).replace(/['"]+/g, '');
+            sex.innerHTML = JSON.stringify(json.entry[i].resource.gender).replace(/['"]+/g, '');
+            birthDate.innerHTML = JSON.stringify(json.entry[i].resource.birthDate).replace(/['"]+/g, '');
+
+            var eachentry = document.createElement('p');
+            eachentry.setAttribute('id', 'resource bundle '+updatedNum);
+            document.getElementById('json').appendChild(eachentry);
+    }
+
+    function json2table(json) {
+        if (chooseFileButtonClicked === true) {
+            json2table_helper(json, 1);
+            chooseFileButtonClicked = false;
+            var display = JSON.stringify(json.entry[1].resource, null, 2);
+            document.getElementById('View_Patient_'+startidx).setAttribute('onclick', 'showPatientEntries('+display+','+startidx+')');
+            startidx++;
+        }
+        else {
+            for (var i=0; i<Object.keys(json.entry).length; i++) {
+                json2table_helper(json, i);
+            }
+            for (var i=startidx; i<Object.keys(json.entry).length; i++) {
+                var display = JSON.stringify(json.entry[i].resource, null, 2);
+                document.getElementById('View_Patient_'+i).setAttribute('onclick', 'showPatientEntries('+display+','+i+')');
+            }
+            startidx += Object.keys(json.entry).length;
+            
+        }
+        
+        
+    }
+
+    function showPatientEntries(display, i) {
+        console.log('show patient entries i='+i);
+        display = JSON.stringify(display, null, 2);
+        document.getElementById('resource bundle '+i).innerHTML = display;
+        document.getElementById('Span_Patient_'+i).innerHTML = "Hide";
+        document.getElementById('View_Patient_'+i).setAttribute('onclick', 'hideEntries('+display+','+i+')');
+    }
+
+    function hideEntries(display, i) {
+        console.log('hide entries i='+i);
+        display = JSON.stringify(display, null, 2);
+        document.getElementById('resource bundle '+i).innerHTML = "";
+        document.getElementById('Span_Patient_'+i).innerHTML = "View";
+        document.getElementById('View_Patient_'+i).setAttribute('onclick', 'showPatientEntries('+display+','+i+')');
+
+    }
+
+
+
+
+    function loadPatients() {
+        myMSALObj.acquireTokenSilent(requestObj).then(function(tokenResponse) {
+            callFHIRServer(FHIRConfig.FHIRendpoint + '/Patient', 'GET', null, tokenResponse.accessToken, FHIRCallback)   
```

Comment:
The `FHIRCallback` here is really just referencing a one-liner higher up in the script, so it really doesn't simplify things. It makes it harder to read to have to scroll up to figure out what is actually going on.